dascore/io/dasdae/core.py

```
@@ -61,9 +59,7 @@ def write(
         spool: SpoolType,
         resource: PyTablesWriter,
         index=False,
-        compression=None,
-        compression_level=None,
-        shuffle=True,
+        storage: DASDAEStorage = DASDAEStorage(),
```

Ruff (0.15.20)

[warning] Do not perform function call `DASDAEStorage` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)